drivers/filesystems/btrfs/btrfs.inf

```diff
@@ -129,7 +129,6 @@ VolumeName              = "Btrfs volume"
 ControllerName          = "Btrfs controller"
 
 [Strings.0404]
-Me                      = "Mark Harmstone"
```

Comment:
This is a third party component

Reply:
Yes, but that duplication is not present upstream. Read the description of my individual commit, where I did this change and I explained that already https://github.com/reactos/reactos/pull/4952/commits/3f57e170fdc73b3e3c90d81c4d66e348f78280f7